code/gateway/dataaccess.q

```diff
@@ -0,0 +1,107 @@
+\d .dataaccess
+
+forceservers:0b
+
+timebarmap:`nanosecond`timespan`microsecond`second`minute`hour`day!1 1 1000 1000000000 60000000000 3600000000000 86400000000000;
```

Comment:
This is duplicated elsewhere, surely doesn't need defined multiple times?

Reply:
Done